src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -56,15 +56,15 @@ public void init() {
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
     response.setContentType("application/json");
-    String user = request.getParameter("user");
+    String countryCode = request.getParameter("countryCode");
 
-    if (user == null || user.equals("")) {
+    if (countryCode == null || countryCode.equals("")) {
```

Comment:
nit: Use countryCode.isEmpty() instead of equals(""). I think it is a clearer way to express what you are trying to check.